frontend/src/Form.jsx

```diff
@@ -4,20 +4,20 @@ import styles from './Form.module.css';
 function Form(props) {
   const [user, setUser] = useState(
     {
-      name: '',
-      pass: '',
+      email: '',
+      password: '',
     },
   );
 
   function handleChange(event) {
     const { name, value } = event.target;
-    if (name === 'pass') setUser({ name: user.name, pass: value });
-    else setUser({ name: value, pass: user.pass });
+    if (name === 'password') setUser({ email: user.email, password: value });
+    else setUser({ email: value, password: user.password });
```

Comment:
Here as well doesnt user store both the name and password? I think this can be cleaned up